app/src/main/java/com/example/habitsmasher/Habit.java

```diff
@@ -119,4 +121,17 @@ public void setDays(String days){
      * @param habitEvents (HabitEventList): The list of habit events to set
      */
     public void setHabitEvents(HabitEventList habitEvents) { _habitEvents = habitEvents; }
+
+    /**
+     * Gets whether the habit is public or private. If true, public. If false, private.
+     * @return true if public, false if private.
+     */
+    @PropertyName("public")
+    public boolean getPublic(){return _public;}
```

Comment:
Similar to the earlier conversation, I'd maybe consider renaming this to isPublic or publicStatus

Reply:
I originally had it named as isPublic, but the firebase really didn't like it. I had to go back and change both the getter and setter to start with "get" and "set". I'll try and change it again and see if it likes it, but if it's not changed then it wouldn't work

Reply:
Ok no worries then